index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Quick startup</title>
+</head>
+
+<body>
+    <!--This wraps on my page together-->
+    <div class="container">
+        <header class="header-page">
+            <!--header contents goes here-->
+
+            <nav>
+                <!--This is my Logo-->
+                <h3>Solorics</h3>
+
+                <!--Toggle button goes here-->
+                <img src="images/Union.png" alt="Toggle_Image_Button" class="toggle">
+            </nav>
+        </header>
+        <main class="main-page">
+            <!--main contents goes here-->
+            <div>
+                <h3>Opeyemi</h3>
+                <h4>Software Developer</h4>
+                <h5>I can help you build a product , feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</h5>
+
+               <hr>
+
+            </div>
+
+
+        </main>
+
+        <div class="section2">
+
+
+            <div class="heading-left">
+                <h4 class="list-heading">Language</h4>
+                <ul>
+                    <li class="items">Javascript</li>
+                    <li class="items">Ruby</li>
+                    <li class="items">Html</li>
+                    <li class="items">Css</li>
+                </ul>
+            </div>
+
+
+
+
+
+            <div class="heading-right">
+                <h4 class="list-heading">Frameworks</h4>
+                <ul>
+                    <li class="items">Bootstrap</li>
+                    <li class="items">Ruby on Rails</li>
+                    <li class="items">RSPec</li>
+                    <li class="items">CapyBara</li>
+                    <li class="items">Selenium</li>
+                </ul>
+
+            </div>
+
+
+
+            <div class="third-heading">
+                <h4 class="list-heading">Skills</h4>
+                <ul>
+                    <li class="items">Codekit</li>
+                    <li class="items">Github</li>
+                    <li class="items">Codepen</li>
+                    <li class="items">Gitlab</li>
+                    <li class="items">Terminal</li>
+                </ul>
+
+            </div>
+        </div>
+
+        <footer class="footer-page">
```

Comment:
- [OPTIONAL] It would be nice to avoid use of `<footer>` tag here , as the `<footer>` tag is meant for the website footer only, not the bottom of the headline section.